cmd/geth/usage.go

```diff
@@ -58,7 +58,6 @@ var AppHelpFlagGroups = []flags.FlagGroup{
 			utils.IdentityFlag,
 			utils.LightKDFFlag,
 			utils.WhitelistFlag,
-			utils.ECBP1100Flag,
```

Comment:
@meowsbits I have also removed ECBP1100 flag as we already transitioned to it. I once thought to leave/keep it, as it's not really a hard-fork.

Reply:
Would it be possible to keep the ECBP1100 flag around, but not touching the genesis/persistent override stuff? 
I'm just thinking about edge cases where it might be really helpful to be able to disable it if needed with `--ecbp1100=9999999999`. Or even different `--artificialfinality` flag? Thoughts?

Reply:
Oh, I forgot that all cases can be configured with `--artificialfinality`. I will bring the flag back, it makes total sense.

Reply:
Resolved with https://github.com/etclabscore/core-geth/pull/367/commits/5cd22b53b2e549a6d15871bc2ca4a05a6297fd1f